post-build/action.yml

```
@@ -57,7 +57,7 @@ runs:
     - name: Count bottles
       id: bottles
       if: always()
-      run: "${{ github.action_path }}/count-bottles.sh" "${{ inputs.debug }}"
+      run: "${ GITHUB_ACTION_PATH }/count-bottles.sh \"${{ inputs.debug }}\""
```

You can avoid the escaping with:

```suggestion
      run: '"${GITHUB_ACTION_PATH}/count-bottles.sh" "${{ inputs.debug }}"'
```